testing/test_pathlib.py

```diff
@@ -89,3 +89,16 @@ def renamed_failed(*args):
     lock_path = get_lock_path(path)
     maybe_delete_a_numbered_dir(path)
     assert not lock_path.is_file()
+
+
+def test_long_path_during_cleanup(tmp_path):
+    """Ensure that deleting long path works (particularly on Windows (#6775))."""
+    path = tmp_path / ("a" * 200)
```

Comment:
If the MAX_PATH on Windows is 260, wouldn't it be better to use e.g. 300 here instead of 200? Or is the test about a path that *expands* to more than 260? If so, should there be an assert that `len(path) > 260`, as a precondition?

Reply:
For the test, it can occur either way. In my failing case, I thought it was important to mention that the path I was using was <260 characters, but would expand to more with the garbage + uuid!

I think that creating a long filename is sufficient here because it covers both cases.

Reply:
I forgot some cases that apply which is mainly that an individual path element, between slashes, can not be greater than 260, so I tried 300 but then switched to using the assert > 260.